waku/v2/protocol/waku_relay.nim

```diff
@@ -114,12 +117,19 @@ method stop*(w: WakuRelay) {.async.} =
   await procCall GossipSub(w).stop()
 
 
-method subscribe*(w: WakuRelay, pubsubTopic: PubsubTopic, handler: SubsciptionHandler|PubsubRawHandler) =
+proc isSubscribed*(w: WakuRelay, topic: PubsubTopic): bool =
+  GossipSub(w).topics.hasKey(topic)
+
+iterator subscribedTopics*(w: WakuRelay): lent PubsubTopic =
+  for topic in GossipSub(w).topics.keys():
+    yield topic
+
+method subscribe*(w: WakuRelay, pubsubTopic: PubsubTopic, handler: WakuRelayHandler) =
```

Comment:
I believe these should be `proc` rather than `method`, as they don't match any base `method`.

Reply:
In line 132, this method calls the parent `subscribe` method:

```Nim
  procCall GossipSub(w).subscribe(pubsubTopic, subsHandler)
```

In one of the follow-up PRs, I rework this implementation with composition instead of inheritance to fix the "baseless methods warning" and other issues.